src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswarningsfilter/InputSuppressWarningsFilter.java

```diff
@@ -1,100 +1,119 @@
+
+com.puppycrawl.tools.checkstyle.checks.naming.MemberNameCheck
+id = ignore
+format = (default)^[a-z][a-zA-Z0-9]*$
+applyToPublic = (default)true
+applyToProtected = (default)true
+applyToPackage = (default)true
+applyToPrivate = (default)true
+
+com.puppycrawl.tools.checkstyle.checks.naming.ConstantNameCheck
+id =
+format = (default)^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$
+applyToPublic = (default)true
+applyToProtected = (default)true
+applyToPackage = (default)true
+applyToPrivate = (default)true
+
+com.puppycrawl.tools.checkstyle.checks.UncommentedMainCheck
+id = ignore
+excludedClasses = (default)^$
+
+com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck
+max = (default)7
+ignoreOverriddenMethods = (default)false
+tokens = (default)METHOD_DEF, CTOR_DEF
+
+com.puppycrawl.tools.checkstyle.checks.coding.IllegalCatchCheck
+illegalClassNames = (default)Error, Exception, RuntimeException, Throwable, java.lang.Error, \
+                    java.lang.Exception, java.lang.RuntimeException, java.lang.Throwable
+
+com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck
+scope = PRIVATE
+excludeScope = (default)null
+skipAnnotations = (default)Generated
+tokens = (default)INTERFACE_DEF, CLASS_DEF, ENUM_DEF, ANNOTATION_DEF, RECORD_DEF
+
+*/
 package com.puppycrawl.tools.checkstyle.filters.suppresswarningsfilter;
-
-/**
- * Test input for using comments to suppress violations.
- *
- * @author Trevor Robinson
- **/
+/** @author Trevor Robinson */
 @SuppressWarnings("foo") // coverage: no following AST
-class InputSuppressWarningsFilter
-{
+class InputSuppressWarningsFilter {
     // AST coverage
-    @SuppressWarnings("foo") interface I { }
-    @SuppressWarnings("foo") enum E { }
+    @SuppressWarnings("foo") interface I { } // violation 'Missing a Javadoc comment'
+    @SuppressWarnings("foo") enum E { } // violation 'Missing a Javadoc comment'
     @SuppressWarnings("foo") InputSuppressWarningsFilter() { }
-    @SuppressWarnings("foo") @interface A { }
+    @SuppressWarnings("foo") @interface A { } // violation 'Missing a Javadoc comment'
 
     // include a non-checkstyle suppression; suppression on same line
-    @SuppressWarnings("unused") private int I; // should fail MemberNameCheck
+    @SuppressWarnings("unused") int I; // violation ''I' must match .* \Q'^[a-z][a-zA-Z0-9]*$'\E'
     @SuppressWarnings({"membername"})
-    private int J; // should NOT fail MemberNameCheck
-    private int K; // should fail MemberNameCheck
+    private int J; // filtered violation 'Name 'J' must match pattern \Q'^[a-z][a-zA-Z0-9]*$'\E'
+    private int K; // violation 'Name 'K' must match pattern \Q'^[a-z][a-zA-Z0-9]*$'\E'
 
     // DO NOT REFORMAT: L and X should be on the same line
     @SuppressWarnings(value="membername")
-    private int L; private int X; // L should NOT fail, X should
+    private int L; private int X; // violation
+    // filtered violation above
```

Comment:
There was some problem adding comments to these violation messages, maybe because of #11075, I will look into it and fix it later.

Reply:
Please share a problem in that ticket